roles/kubevirt/tasks/provision.yml

```diff
@@ -25,44 +25,13 @@
   command: "oc adm policy add-scc-to-user hostmount-anyuid -z kubevirt-infra -n {{ namespace }}"
   when: cluster=="openshift"
 
-- name: Check for kubevirt.yaml template in {{ kubevirt_template_dir }}
-  stat:
-    path: "{{ kubevirt_template_dir }}/kubevirt.yaml"
-  register: byo_template
-
-- name: Check for offline v{{ version }} templates in {{ offline_template_dir }}
-  stat:
-    path: "{{ offline_template_dir }}/v{{ version }}/kubevirt.yaml"
-  register: offline_templates
-  when: byo_template.stat.exists == False
-
-- name: Download KubeVirt Template
-  get_url:
-    url: "{{ release_manifest_url }}/v{{ version }}/kubevirt.yaml"
-    dest: "/tmp/kubevirt.yaml"
-  when: byo_template.stat.exists == False and offline_templates.stat.exists == False
-
-- name: Copy offline templates to /tmp
-  copy:
-    src: "{{ offline_template_dir }}/v{{ version }}/kubevirt.yaml"
-    dest: "/tmp/kubevirt.yaml"
-  when: offline_templates.stat.exists == True
-
-- name: Render KubeVirt Yaml
-  replace:
-    path: "/tmp/kubevirt.yaml"
-    regexp: "namespace: kube-system"
-    replace: "namespace: {{ namespace }}"
-  when: byo_template.stat.exists == False
-
-- name: Render BYO template
-  template:
-    src: "{{ kubevirt_template_dir }}/kubevirt.yaml"
-    dest: /tmp/kubevirt.yaml
-  when: byo_template.stat.exists == True
-
-- name: Create KubeVirt Resources
-  command: kubectl apply -f /tmp/kubevirt.yaml
+- name: Prepare and apply KubeVirt manifests
+  include_tasks: render-and-apply-manifest.yml
+  with_items:
+    - kubevirt.yaml
+    - demo-content.yaml
```

Comment:
From where do we get this file?

Reply:
From KubeVirt release page, or from offline templates. depends on env (online / disconnected)